source/material_model/visco_plastic.cc

```diff
@@ -216,6 +216,17 @@ namespace aspect
               out.thermal_conductivities[i] = MaterialUtilities::average_value (volume_fractions, thermal_conductivities, MaterialUtilities::arithmetic);
             }
 
+          /* In the following, the thermal conductivity is computed, if this option was selected.
+           */
+          if (use_hydrothermal_conductivity == true)
+            {
+              out.thermal_conductivities[i] = out.thermal_conductivities[i] + out.thermal_conductivities[i] *
+                                              (nusselt_number -1) * std::exp(smoothing_factor *
+                                                                             (1 - in.temperature[i] / cutoff_maximum_temperature)) *
```

Comment:
For clarity, would you mind adding a `.` to the `1`s so that they are explicitly doubles? And add a space after the `-` in line 224? That's just to make it match with the rest of the equation :)